lib/config/config_test.go

```diff
@@ -739,6 +739,24 @@ func TestGUIConfigURL(t *testing.T) {
 	}
 }
 
+func TestGUIPasswordHash(t *testing.T) {
+	var c GUIConfiguration
+
+	testPass := "pass"
+	if err := c.SetHashedPassword(testPass); err != nil {
+		t.Fatal(err)
+	}
+
+	if err := c.CompareHashedPassword(testPass); err != nil {
+		t.Errorf("No match on same password: %v", err)
+	}
```

Comment:
Should probably test c.Password != testPass

Reply:
Sorry, I'm confused what you mean.  Compare the hash and the plaintext?

Reply:
If you made HashAndSetPassword and CompareHashedPassword completely noop (ie set the password as plaintext), the test would still pass.

Reply:
Okay understood.